scripts/steam.js

```diff
@@ -148,7 +148,7 @@ module.exports = robot => {
       msg.send(text)
     } else {
       msg.send('Actualmente _Steam_ no responde.')
-      robot.emit('error', err || new Error(`Status code ${res.statusCode}`), msg, 'steam')
+      robot.emit('error', err || new Error(`Status code ${err.statusCode}`), msg, 'steam')
```

Comment:
El `OR` esta debas porque siempre vendrá el `err` dado que solo es llamado en el `catch`

```suggestion
      robot.emit('error', err, msg, 'steam')
```